.github/workflows/ci.yml

```
@@ -26,22 +40,14 @@ jobs:
           --health-retries 5
 
     steps:
+      - uses: actions/checkout@v4
+
       - name: 소스 코드 Checkout
         uses: actions/checkout@v4
-
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)